bundles/core/src/test/resources/form/checkboxgroup/exporter-checkboxgroup.json

```diff
@@ -10,7 +10,8 @@
   "tooltip": "test-short-description",
   "properties": {
     "af:layout" : {
```

Comment:
it should be afs:layout

Reply:
@rismehta @nit23uec

Reply:
I have changed it and pushed it in my branch. @vdua @rismehta @nit23uec